script.js

```diff
@@ -0,0 +1,27 @@
+const openMenuBtn = document.querySelector('.open-menu-svg');
+const closeMenuBtn = document.querySelector('.close-menu-svg');
+const mdHeaderNav = document.querySelector('.small-header-nav');
+const mdHeaderNavLinks = document.querySelectorAll('.md-header-nav-li');
+
+const openMenuBtnFunc = () => {
+  mdHeaderNav.classList.add('d-flex');
+  openMenuBtn.classList.add('d-none');
+};
+
+const closeMenuBtnFunc = () => {
+  mdHeaderNav.classList.remove('d-flex');
+  openMenuBtn.classList.remove('d-none');
+};
+
+for (let i = 0; i < mdHeaderNavLinks.length; i += 1) {
+  mdHeaderNavLinks[i].addEventListener('click', function () {
+    mdHeaderNav.classList.remove('d-flex');
+    openMenuBtn.classList.remove('d-none');
+  });
+}
```

Comment:
Let's change this to use `forEach` rather than for loop, you can use that to be more used to learning the use of higher-order function. Learn more here - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach